src/main/java/by/custom/utilcalculator/service/UserProgressManager.java

```diff
@@ -28,6 +28,18 @@ public void createNewUserProgress(final String chatID) throws UtilsborException
         userProgressStorage.save(userProgress);
     }
 
+    public String processStart(final Command requestingCommand, final String chatID) throws UtilsborException {
+        final UserProgress userProgress;
+        userProgress = userProgressStorage.get(chatID);
+
+        if (!CommandTree.validateCommand(requestingCommand, userProgress)) {
```

Comment:
It's better if boolean is return type.
```suggestion
        if (!CommandTree.isValidCommand(requestingCommand, userProgress)) {
```

Reply:
Done.